go/internal/feast/model/featureview.go

```diff
@@ -30,23 +31,37 @@ func NewFeatureViewFromProto(proto *core.FeatureView) *FeatureView {
 	} else {
 		featureView.Entities = proto.Spec.Entities
 	}
+	entityColumns := make([]*Feature, len(proto.Spec.EntityColumns))
+	for i, entityColumn := range proto.Spec.EntityColumns {
+		entityColumns[i] = NewFeatureFromProto(entityColumn)
+	}
+	featureView.EntityColumns = entityColumns
 	return featureView
 }
 
-func (fs *FeatureView) NewFeatureViewFromBase(base *BaseFeatureView) *FeatureView {
-	ttl := durationpb.Duration{Seconds: fs.Ttl.Seconds, Nanos: fs.Ttl.Nanos}
+func (fv *FeatureView) NewFeatureViewFromBase(base *BaseFeatureView) *FeatureView {
+	ttl := durationpb.Duration{Seconds: fv.Ttl.Seconds, Nanos: fv.Ttl.Nanos}
 	featureView := &FeatureView{Base: base,
 		Ttl:      &ttl,
-		Entities: fs.Entities,
+		Entities: fv.Entities,
 	}
 	return featureView
 }
 
-func (fs *FeatureView) HasEntity(lookup string) bool {
-	for _, entityName := range fs.Entities {
+func (fv *FeatureView) HasEntity(lookup string) bool {
```

Comment:
maybe rename lookup to EntityName?

Reply:
renamed to `name` since `entityName` would conflict with the variable name